packages/openapi-ts/src/types/config.d.ts

```diff
@@ -65,6 +65,18 @@ interface Input {
    * Organization created in Hey API platform.
    */
   organization?: string;
+  /**
+   * Pagination configuration
+   */
+  pagination?: {
+    /**
+     * Array of keywords to be considered as pagination field names.
+     * These will be used to detect pagination fields in schemas and parameters.
+     *
+     * @default ['after', 'before', 'cursor', 'offset', 'page', 'start']
+     */
+    keywords?: string[];
```

Comment:
Let's make this a readonly array for consistency?

```suggestion
    keywords?: ReadonlyArray<string>;
```

Reply:
[602a624](https://github.com/hey-api/openapi-ts/pull/1827/commits/602a6243219d1d3a6506bcc42b003472ade83d89)